packages/redux/src/analytics/middlewares/setUser.ts

```diff
@@ -61,38 +73,42 @@ const getActionTypes = (actionTypes, defaultActionTypes) => {
 };
 
 // Reference to the current user data
-let currentUser;
+let currentUser: Record<string, unknown> | null;
```

Comment:
This type can now be the `UserType` that you have defined. Please, make sure you replace any `Record<string, unknown>` throughout this file with the right types and check if any casts are still necessary after replacing it.
The use of `Record<string, unknown>` should be confined to type aliases and type casts should also be minimised as much as possible.

Reply:
Can you check again?